tests/network_test.cpp

```diff
@@ -44,10 +44,8 @@ struct NetworkTest : BaseTest {};
 // Test creates two Network setup and verifies sending block
 // between is successfull
 TEST_F(NetworkTest, transfer_block) {
-  std::shared_ptr<Network> nw1(
-      new taraxa::Network(g_conf1->network, g_conf1->chain.dag_genesis_block.getHash().toString(), addr_t()));
-  std::shared_ptr<Network> nw2(
-      new taraxa::Network(g_conf2->network, g_conf2->chain.dag_genesis_block.getHash().toString(), addr_t()));
+  std::unique_ptr<Network> nw1(new taraxa::Network(g_conf1->network));
+  std::unique_ptr<Network> nw2(new taraxa::Network(g_conf2->network));
 
```

Comment:
please use make_unique. "New" is not really part of modern c++ anymore...

Reply:
Again, you demand that I refactor the old code. The moral aspect here is: if I do it - I'm a good person, but I'm not required to do it. And I actually do it quite often, but not *always*